Source/WebCore/dom/Document.cpp

```diff
@@ -8992,15 +8993,41 @@ void Document::handlePopoverLightDismiss(PointerEvent& event)
     RefPtr popoverToAvoidHiding = [&]() -> HTMLElement* {
         auto& target = downcast<Node>(*event.target());
         auto* startElement = is<Element>(target) ? &downcast<Element>(target) : target.parentElement();
-        for (auto& element : lineageOfType<HTMLElement>(*startElement)) {
-            if (element.popoverState() != PopoverState::Auto)
-                continue;
-            if (element.popoverData()->visibilityState() != PopoverVisibilityState::Showing)
-                continue;
-            return &element;
-        }
-        // FIXME: Handle popovertarget attributes.
-        return nullptr;
+        auto [clickedPopover, invokerPopover] = [&]() {
+            HTMLElement* clickedPopover = nullptr;
+            HTMLElement* invokerPopover = nullptr;
+            auto isShowingPopover = [](HTMLElement& element) -> bool {
```

Comment:
Note to self: rename to `isShowingAutoPopover` since this intentionally excludes manual popovers.